### PR TITLE
fix extensions docs

### PR DESCRIPTION
**Pull request by fjy**

### CODE REVIEW

File: docs/content/operations/including-extensions.md [hunk 1]
```diff
@@ -1,83 +1,71 @@
...
+```
+druid.extensions.loadList=["postgresql-metadata-storage", "druid-hdfs-storage"]
+```
 
-To let Druid load your extensions, follow the steps below
+These extensions are located in the `extensions` directory of the distribution.
 
-**Tell Druid where your extensions are**
+<div class="note info">
+Druid bundles two sets of configurations: one for the <a href="../tutorials/quickstart.md">quickstart</a> and
```
Comment (gianm) [inline docs/content/operations/including-extensions.md]: quickstart.html

File: docs/content/operations/including-extensions.md [hunk 2]
```diff
@@ -1,83 +1,71 @@
...
-**Tell Druid where your extensions are**
+<div class="note info">
+Druid bundles two sets of configurations: one for the <a href="../tutorials/quickstart.md">quickstart</a> and 
+one for a <a href="../tutorials/cluster.html">clustered configuration</a>. Make sure you are updating the correct 
+common.runtime.properties for your setup.
+</div>
 
-Specify `druid.extensions.directory` (the root directory that contains Druid extensions). See [Configuration](../configuration/index.html)
+<div class="note caution">
+Because of licensing, the mysql-metadata-storage extension is not packaged with the default Druid tarball. In order to get it, you can download it from <a href="http://druid.io/downloads.html">druid.io]</a>,
```
Comment (gianm) [inline docs/content/operations/including-extensions.md]: extra `]`

File: docs/content/operations/including-extensions.md [hunk 3]
```diff
@@ -1,83 +1,71 @@
...
+<div class="note info">
+Druid bundles two sets of configurations: one for the <a href="../tutorials/quickstart.md">quickstart</a> and 
+one for a <a href="../tutorials/cluster.html">clustered configuration</a>. Make sure you are updating the correct 
+common.runtime.properties for your setup.
+</div>
 
-Specify `druid.extensions.directory` (the root directory that contains Druid extensions). See [Configuration](../configuration/index.html)
+<div class="note caution">
+Because of licensing, the mysql-metadata-storage extension is not packaged with the default Druid tarball. In order to get it, you can download it from <a href="http://druid.io/downloads.html">druid.io]</a>, 
+then unpack and move it into the extensions directory. Make sure to include the name of the extension in the loadList configuration.
```
Comment (gianm) [inline docs/content/operations/including-extensions.md]: example would be nice

File: docs/content/toc.md [hunk 4]
```diff
@@ -86,8 +86,7 @@
 ## Development
   * [Overview](../development/overview.html)
   * [Libraries](../development/libraries.html)  
-  * [Extending Druid](../development/modules.html)
```
Comment (gianm) [inline docs/content/toc.md]: I think after this nothing will link to modules.html, is that intentional? Does that doc provide any useful content?

Comment (gianm) [inline docs/content/toc.md]: nm, I see you added a link.

Comment (fjy) [inline docs/content/toc.md]: the extensions doc links to it in the section about how to write your own extension

**Comment by gianm** [comment]
👍

**Pull request merged**